modules/create-new-system-policy.adoc

```diff
@@ -29,7 +29,7 @@ Therefore, include all information explaining the policy.
 +
 [NOTE]
 ====
-You must integrate {product-title} with your notification providers, such as webhooks, Jira, or PagerDuty, to receive alert notifications. Notifiers only show up if you have integrated any notification providers with {product-title}.
+You must integrate {product-title-short} with your notification providers, such as webhooks, Jira, or PagerDuty, to receive alert notifications. Notifiers only show up if you have integrated any notification providers with {product-title}.
```

Comment:
```suggestion
You must integrate {product-title-short} with your notification providers, such as webhooks, Jira, or PagerDuty, to receive alert notifications. Notifications only show up if you have integrated any notification providers with {product-title}.
```

Notifiers correct?

Reply:
A notifier is what is configured to send notifications (using the GUI item "attach notifiers").